packages/dev/core/src/LibDeclarations/webgpu.d.ts

```diff
@@ -977,13 +977,13 @@ declare class GPUError {
     readonly message: string;
 }
 
-declare class GPUOutOfMemoryError implements GPUError {
+declare class GPUOutOfMemoryError extends GPUError {
     private __brand: void;

```

Comment:
should nt we remove the _brand override of a private in case of extend ? similar to the class below ?